packages/language-server/test/completion-items-spec.ts

```diff
@@ -172,7 +180,12 @@ describe("the UI5 language assistant Code Completion Services", () => {
       "upload",
       "util",
       "ux3",
-      "uxap"
+      "uxap",
+      "ubc",
+      "ui",
+      "ui", // This "ui" is in a different parent
```

Comment:
is there more information on the suggestion to distinguish those? 
Basically make an assertion on more then just the label as you have implemented
in other tests?

Reply:
There is, we can add the `detail`. Didn't want to change the tests right now.